### ISSUE TITLE
Extension mistakes second parameter type as another parameter

### ISSUE THREAD

**Issue by jeremyvii** [jeremyvii/vs-docblockr#22]
**Describe the bug**
Extension mistakes second parameter type as another parameter when language is set to TypeScript

**To Reproduce**
1. Attempt to run VS DocBlockr on TypeScript function with two type defined parameters
2. See three parameter tags have been generated

**Expected behavior**
``` typescript
/**
 * [foo description]
 *
 * @param   {number}  bar     [bar description]
 * @param   {string}  thing   [thing description]
 *
 * @return  {void}            [return description]
 */
function foo(bar: number, thing: string): void {

}
```

**Result**
``` typescript
/**
 * [foo description]
 *
 * @param   {number}  bar     [bar description]
 * @param   {[type]}  thing:  [thing: description]
 * @param   {[type]}  string  [string description]
 *
 * @return  {void}            [return description]
 */
function foo(bar: number, thing: string): void {

}
```

**Enviroment (please complete the following information):**
- Version: 1.30.2
- Commit: 61122f88f0bf01e2ac16bdb9e1bc4571755f5bd8
- Date: 2019-01-07T22:49:48.319Z
- Electron: 2.0.12
- Chrome: 61.0.3163.100
- Node.js: 8.9.3
- V8: 6.1.534.41
- OS: Linux x64 4.15.0-43-generic

**Comment by ShaunSpringer** [comment]
Also having this issue recently (dont know it if broke on a recent version or not)

**Comment by jeremyvii** [comment]
This should fixed with the release of [0.4.8](https://github.com/jeremyvii/vs-docblockr/releases/tag/0.4.8).
